src/main/java/com/tradeshift/flagr/Flagr.java

```diff
@@ -82,4 +83,35 @@ public EvaluationResponse evaluate(EvaluationContext context) {
         }
         return evalResponse;
     }
+
+    public Boolean evaluateBoolean(EvaluationContext context) {
+        try {
+            EvaluationResponse evalResponse = this.evaluate(context);
+            String variant = evalResponse.getVariantKey();
+            if (variant != null) {
+                return Boolean.parseBoolean(variant) || variant.toLowerCase().trim().equals("enabled");
+            }
+            return false;
+        } catch (FlagrException e) {
```

Comment:
I don't think we should catch this exception if it means that the flag could not be found.

Reply:
I was imagining those methods as the simplest way of using it. To the point that I would be able to deploy before the flag is in place. But I get the point it can be misleading as well. What about a server error? Should I just throw it on both cases?